jenkins-plugin/pom.xml

```diff
@@ -77,6 +77,7 @@
     <scm-api-plugin.version>2.1.1</scm-api-plugin.version>
     <git-plugin.version>3.3.0</git-plugin.version>
     <junit-plugin.version>1.23</junit-plugin.version>
+    <java.level>7</java.level>
```

Comment:
I thought that Jenkins required Java 8. Am I wrong?

Reply:
Starting with 2.60, yes, but since this uses core 2.46.3, it should still be Java 7.